docs/bit-scope.md

```diff
@@ -46,15 +46,15 @@ As shows in the [configurating bit](configurating-bit.md) section, you can list
 
-## Scope CI
+## Isolated Build/Test
 
-Bit comes with a built-in slim CI mechanism.
+Bit is able to run your components buid and test steps in an isolated environment.
 
-The CI mechanism uses Bit's [environment boilerplating](bit-component.md#component-environment) to create an ad-hock CI process for each code component. This process happens while a component is being committed to the local scope, and exported to a remote scope. 
+This mechanism uses Bit's [environment boilerplating](bit-component.md#component-environment) to create an ad-hock isolated runtime environment for each code component. This process happens while a component is being committed to the local scope, and exported to a remote scope. 
 
 By utilizing each component's build and test environments, Bit has the instructions for building and testing components. Also, if a component has a dependency (or even requires an external package from a known source - NPM for example), Bit will retrieve the dependencies when starting the process. This way Bit makes sure that each component is indeed isolated and encapsulated.
 
-The reason Bit has this built-in CI engine, is that figuring out a proper CI process for packages is a hassle. It requires setting up a build server, figuring out the right scripts and maintaining everything alongside with your package itself. For large packages this makes sense, but when using micro-packages - this is a lot of work. On top of that - it's very hard to update and debug these processes, and even update, if the package itself changes the underline technology it uses (for example - changing the library used to run tests). Bit makes it to be a simple process, which is based of the fact that the environments it uses for build and test are actual code components (which Bit is designed to manage), with a specific API (to run and get results) and an internal dependency management for other components as well as external packages. All are parts of the ecosystem, which together form a CI engine for code components.
+The reason Bit has this isolattion engine, is to validate that a component has all it's runtime and development requirements properly configured. This ease the process of maintaining and using components, because if Bit is able to run it on isolated environment, then if you will use this component in any project, you can rely on Bit to be able to get all the reuired resources. And also, for when you (or other developers) want to make changes to the component and maintaining it, Bit will be able to set up a complete development environment for the components. Bit makes it to be a simple process, which is based of the fact that the environments it uses for build and test are actual code components (which Bit is designed to manage), with a specific API (to run and get results) and an internal dependency management for other components as well as external packages.
```

Comment:
> The reason Bit has this isolattion engine, is to validate that a component has all it's runtime and development requirements properly configured. This ease the process of maintaining and using components, because if Bit is able to run it on isolated environment, then if you will use this component in any project, you can rely on Bit to be able to get all the reuired resources. And also, for when you (or other developers) want to make changes to the component and maintaining it, Bit will be able to set up a complete development environment for the components. Bit makes it to be a simple process, which is based of the fact that the environments it uses for build and test are actual code components (which Bit is designed to manage), with a specific API (to run and get results) and an internal dependency management for other components as well as external packages.

Bit has this **isolation** engine in order to validate the component's configuration of package/component dependencies (make sure it'll function on its own). It fetches the package dependencies (npm modules) and the component dependencies (bit components) in order to assimilate a development environment, it builds and runs the tests, if the test pass it means the dependencies were properly configured.
It will also help other developers to automatically setup a development environment for the component.
Component environments (uses for build and test) are actual code components (managed by bit) with an interface they need to implement. 

(we should also provide a link to a section that describes those interfaces)